templates/named.conf.options.erb

```diff
@@ -1,5 +1,5 @@
 options {
-	directory "/var/cache/bind";
```

Comment:
No, my understanding is that this is the directory where named expects to find all unqualified filenames.  In RedHat, this defaults to `/var/named`, and zone files in `named.conf` are unqualified.  I guess the Debian default is to put the directory option as `/var/cache/named`, and fully-qualify all zone file names to `/etc/db/zones/`_filename_.